Resources/Private/Partials/Backend/Handler/DoktypeDefaultHandler/Node.html

```diff
@@ -12,7 +12,7 @@
 
 
 <f:section name="TableNode">
-<div class="tvp-node sortableItem {f:if(condition: '{settings.configuration.is10orNewer}', then:'', else: 'panel panel-default')} card {node.rendering.stylingClasses}" id="item{node.rendering.md5}" data-record-table="{node.raw.table}" data-record-uid="{node.raw.entity.uid}" data-parent-pointer="{node.rendering.parentPointer}">
+<div class="tvp-node sortableItem {f:if(condition: '{settings.configuration.is10orNewer}', then:'', else: 'panel panel-default')} card {node.rendering.stylingClasses}" id="item{node.rendering.md5}" data-record-table="{node.raw.table}" data-record-uid="{node.raw.entity.uid}" data-parent-pointer="{node.rendering.parentPointer}" {node.rendering.additionalRecordData -> f:format.raw()}>
```

Comment:
Who will consume this extra data?

Reply:
Sitepackages reacting on that. 

We spice the node design via CSS to e.g. incorporate background container colours, parallax images etc. So this is not a different type of preview node type, just some attribute.
The original PR for v7 doing that just for tt_content was https://github.com/T3Voila/templavoilaplus/pull/182 so it is just about restoring the feature.